Riot/Modules/Room/RoomViewController.m

```diff
@@ -1534,39 +1536,49 @@ - (void)isDirectChatLeftByTheOther:(void (^)(BOOL isEmptyDirect))onComplete
                     MXMembership directUserMembership = directUserMember.membership;
                     if (directUserMembership != MXMembershipJoin && directUserMembership != MXMembershipInvite)
                     {
-                        onComplete(YES);
+                        onComplete(YES, nil);
                     }
                     else
                     {
-                        onComplete(NO);
+                        onComplete(NO, nil);
                     }
                 }
                 else
                 {
                     MXLogDebug(@"[RoomViewController] isEmptyDirectChat: the direct user has disappeared");
-                    onComplete(YES);
+                    onComplete(YES, nil);
                 }
             } failure:^(NSError *error) {
                 MXLogDebug(@"[RoomViewController] isEmptyDirectChat: cannot get all room members");
-                onComplete(NO);
+                [[AppDelegate theDelegate] showErrorAsAlert:error];
```

Comment:
you can remove this line, because the new completion block handle the error
(or remove `[self showError:error];` from the completion block)

Reply:
Fixed by last commit.